### PR TITLE
refactor(btc|ckb|examples): Update examples with rgbpp package and env variables

### PR DESCRIPTION
**Pull request by duanyytop**
### Examples

- Split examples to two parts:  rgbpp(RGB++ xUDT and Spore) and xudt(xUDT on CKB) examples
- Use env variables instead of constants for examples
- Add env description and remove local examples description in README
- It is recommended to use Queue service to implement RGB++ transfer and unlock BTC time cells
- Use rgbpp package instead of sub packages(btc/ckb/service)

### BTC package

- Export `ECPairInterface` from btc package for public environment variables

### CKB package

- Export `RawClusterData` and `RawSporeData` from ckb package to avoid re-importing `spore-sdk/core` for examples

### CODE REVIEW

File: examples/rgbpp/.env.example [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CKB_INDEXER_URL=https://testnet.ckb.dev/indexer
+
+
+# BTC Variables
+
+# The BTC private key whose format is 32bytes hex string without 0x prefix
+BTC_PRIVATE_KEY=private-key
+
+# The BTC assets api url which should be matched with IS_MAINNET
+VITE_BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro;
```
Comment (ShookLyngs) [inline examples/rgbpp/.env.example:22]: Is the ";" at the end valid? Same question for below variables.
```suggestion
VITE_BTC_SERVICE_URL=https://btc-assets-api.testnet.mibao.pro
```

**Review by ShookLyngs — CHANGES_REQUESTED** [review]

File: examples/rgbpp/.env.example (hunk 1 above)
Comment (duanyytop) [reply examples/rgbpp/.env.example]: removed

**Review by ShookLyngs — APPROVED** [review]

File: examples/xudt/.env.example [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Ture for CKB Mainnet and false for CKB Testnet, the default value is false
```
Comment (Flouse) [inline examples/xudt/.env.example:1]:
```suggestion
# True for CKB Mainnet and false for CKB Testnet, the default value is false
```

File: examples/rgbpp/README.md [hunk 3]
```diff
@@ -1,147 +1,170 @@
...
+npx ts-node xudt/launch/2-launch-rgbpp.ts
 ```
-
-### Leap RGB++ xUDT from BTC to CKB
+### 3. Distribute RGB++ xUDT on BTC
 
 ```shell
-npx ts-node examples/rgbpp/queue/3-btc-jump-ckb.ts 
+npx ts-node xudt/launch/3-distribute-rgbpp.ts
 ```
```
Comment (Flouse) [inline examples/rgbpp/README.md:66]:
```suggestion
#### 1. Prepare Launch

```shell
npx ts-node xudt/launch/1-prepare-launch.ts
```
#### 2. Launch RGB++ xUDT on BTC

```shell
npx ts-node xudt/launch/2-launch-rgbpp.ts
```
#### 3. Distribute RGB++ xUDT on BTC

```shell
npx ts-node xudt/launch/3-distribute-rgbpp.ts
```

```

**Review by Flouse — APPROVED** [review]

**Comment by duanyytop** [comment]
@Flouse All the above comments have been updated in https://github.com/ckb-cell/rgbpp-sdk/pull/159/commits/ec6fde2d6b8478cd2459c8e0c7cfb703d4ad58f9

**Comment by Dawn-githup** [comment]
https://inquisitive-puma-8a7.notion.site/Update-examples-with-rgbpp-package-and-env-variables-test-5813201894e1413ea2ba69a30bdb1587

**Review by Dawn-githup — APPROVED** [review]

**Pull request merged**